blant.c

```diff
@@ -1173,6 +1178,103 @@ void ProcessGraphlet(GRAPH *G, SET *V, unsigned Varray[], char perm[], TINY_GRAP
 	}
 }
 
+static int nChoosek(int n, int k) {
+	if(k>=n) return 0;
+	int a = 1, b = 1, i, j;
+	for(i=n; i >=n-k+1; i--) a*=i;
+	for(j=k; j>=1; j--) b*=j;
+	return a/b;
+}
+
+// Right now use least frequent windowRep canonicals 
+int FindWindowRepInWindow(GRAPH *G, SET *W, int (*windowReps)[_k], int *windowRepInt)
+{
+	int WArray[_windowSize];	// Window node array
+	Boolean isconnected = false; //Check whether there is any windowRep in the window
+	assert(SetToArray(WArray, W) == _windowSize);
+	int ca[_k], i, j, Gint, GintCanon, GintCanonInt;
+	MULTISET* ordinalMSET = MultisetAlloc(pow(2, _k * (_k - 1) / 2));
+	COMBIN *c = CombinZeroth(_windowSize, _k, ca);	// (W choose K) many k-node graphlets in Window
```

Comment:
For now this is fine but we may also need to look into faster methods, eg traversing only the connected parts.